src/lists/views/list/networks-search-result-table-template.html

```diff
@@ -1,117 +1,114 @@
-                        </p>
-                        <a href="{{'network/' + summary.id | getBaseUrl}}" class="read-more-btn">{{"read-more" | translate}}</a>
-                        <div class="clear-fix"></div>
-                        <div class="sm-top-margin countDetail">
-                            {{counts=summary['obiba.mica.CountStatsDto.networkCountStats'];""}}
-                            <a ng-if="counts.individualStudies"
-                               href="{{'studies' | doSearchQuery:'network(in(Mica_network.id,' + summary.id +  ')),study(in(Mica_study.className,Study))' }}"
-                               class="btn btn-default btn-xxs"
-                               test-ref="individualStudyCount">
-                                <localized-number
-                                        value="counts.individualStudies"></localized-number>
-                                {{counts.individualStudies > 1 ?
-                                "global.individual-studies":"global.individual-study"
-                                | translate}}
-                            </a>
+                    <img src="{{summary.logoUrl}}"
+                         class="img-responsive"/>
 
-                            <a ng-if="counts.studiesWithVariables"
-                               href="{{'studies' | doSearchQuery : 'network(in(Mica_network.id,' + summary.id +  ')),variable(in(Mica_variable.variableType,Collected))' }}"
-                               class="btn btn-default btn-xxs"
-                               test-ref="studyWithVariablesCount">
-                                <localized-number
-                                        value="counts.studiesWithVariables"></localized-number>
-                                {{counts.studiesWithVariables > 1 ?
-                                "metrics.mica.studies-with-variables" :
-                                "metrics.mica.study-with-variables"
-                                | translate}}
-                            </a>
+                    <h1 ng-if="!summary.logo" src="" alt=""
+                        class="big-character">
+                        <span class="t_badge color_light i-obiba-N"></span>
+                    </h1>
+                </div>
+                <div class="col-md-10  col-sm-12 col-xs-12">
+                    <h4>
+                        <a href="{{'network/' + summary.id | getBaseUrl}}">
+                            <localized value="summary.name"
+                                       lang="lang"></localized>
+                        </a>
+                    </h4>
+                    <p>
+                        <localized value="summary.description" lang="lang"
+                                   ellipsis-size="250"
+                                   markdown-it="true"></localized>
+                    </p>
+                    <a href="{{'network/' + summary.id | getBaseUrl}}" class="read-more-btn">{{"global.read-more" | translate}}</a>
```

Comment:
I still expect a "label" style...

Reply:
The read more have to be a link in default Drupal theme, but it may be a button (btn) in other template, but it does not be an label style

Reply:
yes